frontend/src/pages/projects/screens/projects/ProjectListView.tsx

```diff
@@ -86,9 +86,10 @@ const ProjectListView: React.FC<ProjectListViewProps> = ({
             <NewProjectButton />
           </ToolbarItem>
           <ToolbarItem>
-            <Button variant="link">
-              <Link to="/notebookController">Launch Jupyter</Link>
-            </Button>
+            <Button
+              variant="link"
+              component={() => <Link to="/notebookController">Launch Jupyter</Link>}
```

Comment:
does this have to be a function or can component just be
```tsx
...
component={<Link to="/notebookController">Launch Jupyter</Link>}
...
```

Reply:
Good question! [A router link needs a function passed](https://www.patternfly.org/v4/components/button#router-link).